bin/common.sh

```diff
@@ -68,7 +68,7 @@ if [[ -f "${ZEPPELIN_CONF_DIR}/zeppelin-env.sh" ]]; then
   . "${ZEPPELIN_CONF_DIR}/zeppelin-env.sh"
 fi
 
-ZEPPELIN_CLASSPATH+=":${ZEPPELIN_CONF_DIR}"
+ZEPPELIN_CLASSPATH=":${ZEPPELIN_CONF_DIR}"
```

Comment:
I think it's okay to keep '+=' to let user define who's additional classpath in zeppelin-env.sh by defining ZEPPELIN_CLASSPATH.